### ISSUE TITLE
Issue about reimplementing medical ZSAD performance

### ISSUE THREAD

**Issue by kiyoonjeong0305** [zqhang/AnomalyCLIP#41]
Hello @zqhang and @tianyu0207,

First of all, I really appreciate for your work! it's really nice and interesting.

But I can't find some implementation details about medical ZSAD.

The paper said "To address this issue, we create such a dataset based on ColonDB (More details see Appendix A), and then optimize the prompts in AnomalyCLIP and VAND using this dataset and evaluate their performance on the medical image datasets.".

So as far as i understood, you finetuned the models with (ColonDB + test split of Endo classfication dataset) and tested medical ZSAD performance of every other medical dataset.
Then what is the finetuning dataset for testing ColonDB and Endo dataset?

Since those two datasets were already used in finetuning the models, I think another dataset is needed for testing those two dataset.

I would appreciate every help

**Comment by zqhang** [comment]
Hi, your understanding is correct. Therefore, we do not test ColonDB and Endo (classification).

**Comment by AlirezaSalehy** [comment]
But you have reported the anomaly segmentation performance on ColonDB, what was your approach? did for example trained the model on CilinicDB + Endo_Cls and then tested on ColonDB?

I would deeply appreciate it if you could make this process clear.

**Comment by AlirezaSalehy** [comment]
But you have reported the anomaly segmentation performance on ColonDB, what was your approach? did for example trained the model on CilinicDB + Endo_Cls and then tested on ColonDB?

I would deeply appreciate if you could make this process clear.

**Comment by AlirezaSalehy** [comment]
In fact label images in EndoTect dataset is around 10K. And i was not able to find a split which contains 163 normal samples according to Appendix A of the paper. 
"we choose the test split
of Endo classification dataset (Hicks et al., 2021) to combine with ColonDB. As a result, the new
medical dataset contains 163 normal samples and 380 anomaly samples, supporting both anomaly
classification and segmentation tasks."
